R/compose_ggplot.R

```diff
@@ -168,6 +168,17 @@ compose_ggplot <- function(
       size = .25
     )
   }
+  # PLOT IMAGES
+  if (nrow(geom_positions$img_data) > 0) {
+    for(r in 1:nrow(geom_positions$img_data)) {
```

Comment:
Use `seq_len()` instead of `1:nrow(...)`

```suggestion
    for (r in seq_len(nrow(geom_positions$img_data))) {